test/unit/spec/common/cellComponents/filePathWidget-Spec.js

```diff
@@ -105,10 +105,10 @@ define([
                     parameters: parameters,
                 })
                 .then(() => {
-                    let preClickNumberOfRows = $('tr').length;
+                    let preClickNumberOfRows = $(node).find('tr').length;
```

Comment:
Something was hanging in the dom after each test. I needed to specify the node so Karma didn't get confused when running all tests.